flowetl/etl/etl/model.py

```diff
@@ -98,3 +98,75 @@ def can_process(cls, *, cdr_type: str, cdr_date: pendulumDate, session: Session)
             process = False
 
         return process
+
+
+# pylint: disable=too-few-public-methods
+class ETLPostQueryOutcome(Base):
+    """
+    DB Model for storing the post etl queries outcome.
+    """
+
+    __tablename__ = "post_etl_queries"
+    __table_args__ = {"schema": "etl"}
+
+    id = Column(Integer, primary_key=True)
+    cdr_type = Column(String)
+    cdr_date = Column(Date)
+    type_of_query_or_check = Column(String)
+    outcome = Column(String)
+    optional_comment_or_description = Column(String)
+
+    def __init__(
+        self,
+        *,
+        cdr_type: str,
+        cdr_date: pendulumDate,
+        type_of_query_or_check: str,
+        outcome: str,
+        optional_comment_or_description: str = "",
+    ):
+
+        self.cdr_type = etl_utils.CDRType(cdr_type)
+        self.cdr_date = cdr_date
+        self.type_of_query_or_check = type_of_query_or_check
+        self.outcome = outcome
+        self.optional_comment_or_description = optional_comment_or_description
+
+    @classmethod
+    def set_outcome(
+        cls,
+        *,
+        cdr_type: str,
+        cdr_date: pendulumDate,
+        type_of_query_or_check: str,
+        outcome: str,
+        optional_comment_or_description: str = "",
+        session: Session,
+    ) -> None:
+        """
+        Add new row to the post-etl book-keeping table.
+
+        Parameters
+        ----------
+        cdr_type : str
+            CDR type of file being processed ("calls", "sms", "mds" or "topups")
```

Comment:
Since flowetl now allows ingesting data from different sources (files and postgres tables), would be good to rephrase the docstring to reflect this. (Also applies below.)
```suggestion
           CDR type of data being processed ("calls", "sms", "mds" or "topups")
```